README.md

```diff
@@ -91,7 +107,10 @@ Define a function called `doSomething`. It should take one argument, called
 
 Your Answer:
 ```js
-// write code here
+doSomething : function(thingToDo){
+  thingToDo(thingToDo);
+}
+doSomething('punch')
```

Comment:
when you call `doSomething` make sure to pass in a reference to a function or an anonymous function